go/pkg/diskcache/sys_windows.go

```diff
@@ -0,0 +1,18 @@
+// System utilities that differ between OS implementations.
+package diskcache
+
+import (
+	"os"
+	"syscall"
+	"time"
+)
+
+// This will return correct values only if `fsutil behavior set disablelastaccess 0` is set.
+// Tracking of last access time is disabled by default on Windows.
+func GetLastAccessTime(path string) (time.Time, error) {
```

Comment:
I'd recommend instead `func FileInfoToAccessTime(i *os.FileInfo) time.Time` and keep the os.Stat() call at the call site.

Reply:
Sure, although note how the Darwin interface needs to differ from the others. Fortunately, the whole mess compiles...